src/components/side_panel/components/checkbox/checkbox.xml

```diff
@@ -0,0 +1,14 @@
+<templates>
+  <t t-name="o-spreadsheet.Checkbox">
+    <label class="o-checkbox">
```

Comment:
should move relevant css inside the component (o-checkbox). Also the case for some other components (o-section, ...)

Reply:
Would need a little rework of the css (🍝 ) so I will do it after this task :) 
(3649539)